lms/static/lms/js/spec/main.js

```diff
@@ -676,6 +676,7 @@
         'course_bookmarks/js/spec/bookmark_button_view_spec.js',
         'course_bookmarks/js/spec/bookmarks_list_view_spec.js',
         'course_bookmarks/js/spec/course_bookmarks_factory_spec.js',
+        'course_search/js/spec/course_search_spec.js',
```

Comment:
This is a RequireJS configuration file, so course experience isn't listed because it is built via Webpack.